src/plugins/unified_search/public/query_string_input/query_bar_menu_panels.tsx

```diff
@@ -297,7 +301,8 @@ export function QueryBarMenuPanels({
   };
 
   const onSelectLanguage = (lang: string) => {
-    http.post('/api/kibana/kql_opt_in_stats', {
+    http.post('/internal/kibana/kql_opt_in_stats', {
```

Comment:
This should be `/internal/kql_opt_in_stats` right? We have removed kibana from the url